modules/navier_stokes/doc/content/source/materials/NSFVMixtureMaterial.md

```diff
@@ -0,0 +1,22 @@
+# NSFVMixtureMaterial
```

Comment:
Especially given that the other non-FV-kernel objects introduced in this PR do not have FV included in the prefix, I think we can also change this name to just have the NS prefix. I appreciate the naming generality of the other objects and I think we can apply that here as well.